src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1670,7 +1668,6 @@ class ContentExplorer extends Component<Props, State> {
                             isMedium={isMedium}
                             isSmall={isSmall}
                             isTouch={isTouch}
-                            metadataColumnsToShow={metadataColumnsToShow}
```

Comment:
`metadataColumnsToShow` prop is no longer needed as the fields to show will be included as part of new metadata query syntax. Use `fields` clause in the query as shown in this example: https://developer.box.com/guides/metadata/quick-start/create-query/